[2025-04-23T23:07:50.500+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T23:07:50.506+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spotify_etl_dag.load_spotify_csv_to_snowflake manual__2025-04-23T23:07:49.751485+00:00 [queued]>
[2025-04-23T23:07:50.508+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spotify_etl_dag.load_spotify_csv_to_snowflake manual__2025-04-23T23:07:49.751485+00:00 [queued]>
[2025-04-23T23:07:50.509+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-04-23T23:07:50.512+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_spotify_csv_to_snowflake> on 2025-04-23 23:07:49.751485+00:00
[2025-04-23T23:07:50.515+0000] {standard_task_runner.py:72} INFO - Started process 187 to run task
[2025-04-23T23:07:50.516+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spotify_etl_dag', 'load_spotify_csv_to_snowflake', 'manual__2025-04-23T23:07:49.751485+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/spotify_etl_dag.py', '--cfg-path', '/tmp/tmprlt0y756']
[2025-04-23T23:07:50.518+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask load_spotify_csv_to_snowflake
[2025-04-23T23:07:50.534+0000] {task_command.py:467} INFO - Running <TaskInstance: spotify_etl_dag.load_spotify_csv_to_snowflake manual__2025-04-23T23:07:49.751485+00:00 [running]> on host fb7f0aec2fef
[2025-04-23T23:07:50.561+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spotify_etl_dag' AIRFLOW_CTX_TASK_ID='load_spotify_csv_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T23:07:49.751485+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T23:07:49.751485+00:00'
[2025-04-23T23:07:50.561+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-23T23:07:50.591+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-23T23:07:50.591+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.10.14, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-23T23:07:50.592+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-23T23:07:50.592+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-23T23:07:51.357+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-23T23:11:27.399+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-04-23T23:11:27.402+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-23T23:11:27.404+0000] {process_utils.py:132} INFO - Sending 15 to group 187. PIDs of all processes in the group: [187]
[2025-04-23T23:11:27.404+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 187
[2025-04-23T23:11:27.405+0000] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-04-23T23:11:27.406+0000] {connection.py:788} INFO - closed
[2025-04-23T23:11:27.535+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-04-23T23:11:27.626+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-23T23:11:27.675+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=187, status='terminated', exitcode=0, started='23:07:50') (187) terminated with exit code 0
